app/main.py

```diff
@@ -158,13 +165,29 @@
     'CACHE_DEFAULT_TIMEOUT': 60
 })
 
+# ----------------------------------------------------------------------------------------------------------------------
```

Comment:
I would rather define this function in `utils.py` and simply import it in `main.py`, what do you think?

Reply:
Done